src/main/webapp/jsp/feedbackSubmissionEdit.jsp

```diff
@@ -134,7 +134,9 @@
                         %>
                         </select>
                     </div>
-                    <div <%=(question.isRecipientNameHidden()) ? "class=\"col-sm-12\"" : "class=\"col-sm-10\""%>>
+                    <div 
+                    <% if (question.isRecipientNameHidden()) out.println("class=\"col-sm-12\""); else if(numOfResponseBoxes == maxResponsesPossible) out.println("class=\"col-sm-10\"");
```

Comment:
Don't omit the curly brackets, even if it's only one statement. Consider splitting this line up into multiple lines.